nano/rpc_test/rpc.cpp

```diff
@@ -2159,11 +2159,11 @@ TEST (rpc, peers)
 	ASSERT_EQ (200, response.status);
 	auto & peers_node (response.json.get_child ("peers"));
 	ASSERT_EQ (2, peers_node.size ());
-	ASSERT_EQ (std::to_string (nano::protocol_version), peers_node.get<std::string> ("UDP: [::1]:24001"));
+	ASSERT_EQ (std::to_string (nano::protocol_version), peers_node.get<std::string> ("[::1]:24001"));
```

Comment:
We can push this change in by itself to make the patch simpler.

Reply:
You mean separate PR?

Reply:
Now part of https://github.com/nanocurrency/nano-node/pull/1971